horizon/authentication.py

```diff
@@ -1,35 +1,30 @@
+from typing import Annotated
+
 from fastapi import Header, HTTPException, status
 
 from horizon.config import MOCK_API_KEY, sidecar_config
 from horizon.startup.api_keys import get_env_api_key
 
 
-def enforce_pdp_token(authorization=Header(None)):
+def enforce_pdp_token(authorization: Annotated[str | None, Header()]):
```

Comment:
are you sure Header() and Header(None) ?

Reply:
Same as here in the previous PR
![image](https://github.com/user-attachments/assets/8924e5b6-46a5-4804-be04-ca7d05cad8bd)

https://github.com/permitio/PDP/pull/226/files#diff-235fdf0e84e00c3e05e9e00d615411c1fde6804276cc5c8a2552bedc582c152eR649-R652

And in FastAPI docs example
https://fastapi.tiangolo.com/tutorial/header-params/?h=header#import-header
```python
from typing import Annotated

from fastapi import FastAPI, Header

app = FastAPI()


@app.get("/items/")
async def read_items(user_agent: Annotated[str | None, Header()] = None):
    return {"User-Agent": user_agent}

```